Dockerfile.tmpl

```diff
@@ -573,6 +570,9 @@ RUN pip install pytorch-ignite \
         timm \
         torchinfo && \
         pip install git+https://github.com/facebookresearch/segment-anything.git && \
+    # b/343971718: remove duplicate aiohttp installs, and reinstall it
+    rm -rf /opt/conda/lib/python3.10/site-packages/aiohttp* && \
+    pip install aiohttp && \
```

Comment:
One thing you'll have to watch out for is that pip does not care about installing something compatible unless its all in "one install". So this is going to try and install the latest aiohttp.

You could try installing with something like mamba which does check against other installed packages, or we could accept that hopefully latest aiohttp is compatible with our image.

Reply:
sounds good switched over to mamba. works the same.